### PR TITLE
Add hacks for BHNT#47 (closes #21)

### LINKED ISSUE #21: Add list of hacks for BHNT#47
Sources:

* https://www.meetup.com/Berlin-Hack-and-Tell/messages/boards/thread/50244682
* https://www.meetup.com/Berlin-Hack-and-Tell/events/234990104/

### PR DESCRIPTION
**Pull request by vog**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,16 @@ Last Tuesday on each(~) month | [Meetup](https://www.meetup.com/de-DE/Berlin-Hac
 
 This event was mislabeled \#46 on Meetup.
 
+* Just in time C++ by Gaetano
+* Plotter - raspberry pi sticker printer by Michael
```
Comment (ligi) [inline README.md]: AFAIR this was @michaelkreil - perhaps he has some details / links?

Comment (ligi) [inline README.md]: will merge anyway - if he has an addition he could add this as a new PR

Comment (MichaelKreil) [inline README.md]: I only have this undocumented repo: https://github.com/dsstio/plotter

Comment (vog) [inline README.md]: @MichaelKreil See #71

**Pull request merged**